cmd/root.go

```diff
@@ -111,6 +115,67 @@ func args() []string {
 	return args
 }
 
+func bugReport(ctx context.Context, version string) error {
+	var commonSocketPaths = []string{
+		"/var/run/docker.sock",
+		"/var/run/podman/podman.sock",
+		"$HOME/.colima/docker.sock",
+		"$XDG_RUNTIME_DIR/docker.sock",
+	}
+
+	sprintf := func(key, val string) string {
+		return fmt.Sprintf("%s:\t\t%s\n", key, val)
+	}
+
+	report := sprintf("act version", version)
+	report += sprintf("GOOS", runtime.GOOS)
+	report += sprintf("GOARCH", runtime.GOARCH)
+	report += sprintf("NumCPU", fmt.Sprint(runtime.NumCPU()))
+
+	if os.Getenv("DOCKER_HOST") == "" {
```

Comment:
We're printing all found sockets in case `DOCKER_HOST` is not set, but shouldn't we also print the value of `DOCKER_HOST` in case it is set?

Reply:
I can't think of any benefit it will provide us or the user, if `DOCKER_HOST` is set then all is fine